content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/moves_history.rst

```diff
@@ -0,0 +1,99 @@
+=============
+Moves history
+=============
+
+The *moves history* report in Odoo *Inventory* provides a detailed record of product movements
```

Comment:
You might have already considered this, but should the report name be capitalized?

Reply:
```suggestion
The *Moves History* report in Odoo *Inventory* provides a detailed record of product movements
```
```suggestion
The *moves history* report in Odoo *Inventory* provides a detailed record of product movements
```